packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1069,6 +1106,7 @@ class DataTable extends StatelessWidget {
         clipBehavior: clipBehavior,
         child: Table(
           columnWidths: tableColumns.asMap(),
+          defaultVerticalAlignment: TableCellVerticalAlignment.middle,
```

Comment:
What does this change do?

Reply:
Without the vertical alignment middle, if there is a table with 2 columns and the first column has one row of text and the second column has 10 rows of text then the text in the first column would appear at the top. See https://github.com/flutter/flutter/issues/70510#issuecomment-871105093

Note that the checkbox is in the middle but not the text in the first and second column of the screenshot in the link.

A user suggested https://github.com/flutter/flutter/issues/70510#issuecomment-950496193 to use `defaultVerticalAlignment: TableCellVerticalAlignment.middle` so that the content is always in the middle.